### DISCUSSION TITLE
Subtabs for bank tag layouts

### DISCUSSION THREAD

**Discussion by Samilton-dev** [runelite/runelite#16975] [Ideas]
When you select a bank tab there will be subtabs to help organize variance for the type of tab you made. Instead of having 4 tabs for tob roles, you can have 4 subtabs under your tob tab. You can have subtabs for bandos for different methods as well.

Currently, I duplicate items 10s of times under a tab for alternate methods but this should help keep things separate and organized. Thoughts?

**Comment by geheur** [comment]
what does a subtab look like?

**Reply by Samilton-dev** [reply]
I imagine it would look like an additional bar along the top bottom or maybe side of the bank UI with room for text or an icon, probably smaller than the current bank tag icons. 

Upon clicking it, it would switch to a blank layout
